playbooks/digitalocean.yml

```diff
@@ -46,7 +46,7 @@
       private: no
 
     - name: "do_access_token_entry"
-      prompt: "\n\nNew API access tokens can be generated in the DigitalOcean control panel.\nhttps://cloud.digitalocean.com/settings/applications\n  * When generating a new token, please note that the Write Scope is necessary\n    in order to create Droplets.\n\nIf this field is left blank, the environment variable DO_API_KEY\nwill be used.\n\nWhat is your DigitalOcean Access Token?\n"
+      prompt: "\n\nPersonal Access Tokens allow Streisand to create a droplet for you.\nNew Personal Access Tokens can be generated in the DigitalOcean control panel.\nTo generate a new token please do the following:\n  * Go to https://cloud.digitalocean.com/settings/applications\n  * Click 'Generate New Token'\n  * Give the token a name (it is arbitrary)\n  * Be sure to select the 'Write' scope as well (this is not optional)\n  * Click 'Generate Token'\n  * Copy the long string that is generated and paste it below.  \n\nIf this field is left blank, the environment variable DO_API_KEY\nwill be used.\n\nWhat is your DigitalOcean Personal Access Token?\n"
```

Comment:
@Frichetten This diff looks good but I'd love if we could also take this as a chance to fix the long line. 

I _think_ you could do something like:
```
prompt: |


Personal Access Tokens allow Streisand to create a droplet for you.
New Personal Access Tokens can be generated in the DigitalOcean control panel.
....
```

This is the "Literal Block Scalar" notation described in [the Ansible YAML docs](https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html). You'd want literal (`|`) and not folded (`>`) in this case to preserve the newlines.

Are you open to giving that a shot? Thanks!